GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -98,6 +98,10 @@ private extension EntryWidget {
                 }
             }
         }
+        if !environment.isAuthenticated() {
```

Comment:
I think this can be covered with test.

Reply:
tests added [059deb7](https://github.com/salemove/ios-sdk-widgets/pull/1091/commits/059deb77207eb3de7d86317697927d8a54e47970)